doc/source/conf.py

```diff
@@ -41,6 +41,11 @@
 import sunpy.data
 sunpy.data.download_sample_data(overwrite=False)
 
+# -- Shut up numpy warnings from WCSAxes
+
+import numpy as np
+np.seterr(invalid='ignore')
```

Comment:
well it doesn't fill the log with as many warnings! There seems to be a couple of transient failures introduced by the timeseries gallery examples, but I think that's because of the web sites it's downloading from.